app/controllers/application.js

```diff
@@ -200,6 +200,22 @@ export default class ApplicationController extends Controller.extend(
     return values.every(({ changed }) => changed === false);
   }
 
+  @tracked
+  openModal = !window.localStorage.hideMessage;
+
+  @tracked
+  dontShowModalAgain = false;
+
+  @action toggleModal() {
+    this.openModal = !this.openModal;
+    // Uncomment this when done so that the message only shows the first time
```

Comment:
Can we remove these 3 commented lines now? I'd rather they not land in develop

Reply:
I've removed them, although I don't really understand why.  They make it easy for any developer in the future to switch between showing the modal once, showing the modal until the user hits the checkbox, or always showing the modal.  What's the drawback?